google/cloud/forseti/notifier/notifiers/slack_webhook.py

```diff
@@ -88,4 +94,7 @@ def run(self):
 
         for violation in self.violations:
             webhook_payload = self._compose(violation=violation)
-            self._send(payload=webhook_payload)
+            try:
+                self._send(payload=webhook_payload)
+            except Exception:  # pylint: disable=broad-except
+                LOGGER.exception('Post was not sent to Slack webhook.')
```

Comment:
"Failed to send payload to Slack webhook." is a better error message. Post is a request type and not the information that's being sent.

Reply:
Post here means message, not HTTP POST method. This is in the same context as `Sends a post to a Slack webhook url` in the `_send` method. Updated. Thanks for pointing this out.